shared/fs/common/path-item-action/layout.js

```diff
@@ -72,7 +72,7 @@ const getRawLayout = (
         ...empty,
         ...(mode === 'screen'
           ? {
-              newFolder: pathItem.type === 'folder',
+              newFolder: pathItem.writable && pathItem.type === 'folder',
```

Comment:
`pathItem.type === 'folder'` shouldn't actually be necessary as we know for sure a TLF root is a folder; Not sure why I did that in the first place; maybe copy/paste laziness. But not a big deal / harmless.

Reply:
Also I think https://github.com/keybase/client/pull/17235/files#diff-924982ffd1e49c940aa1080b466eb57fR92 needs this too (where the `type === 'folder'` is needed).